docs/about/changelog.md

```diff
@@ -37,6 +37,10 @@
       debugging instead of the Templight dump. This makes it possible to work
       with a Clang binary (patched with Templight) and does not require a
       Templight binary. Note that profiling is not supported.
+    * When a template is instantiated in the environment (eg. by files included
+      into the shell) and then an instantiation attempt of the same template
+      with the same arguments is debugged with mdb, only the memoization is
+      shown and the original instantiation is not displayed in the debugger.
```

Comment:
This might be a usability regression. I think it's common to include a whole cpp into a metashell session which already instantiates the problematic template, which is then later debugged. Can we provide some kind of workaround proposal for the user if this is the case?

One workaround could be to debug the whole environment, and inserting a breakpoint for the template in question. This approach could become difficult if the original name of the type is hidden under multiple layers of typedefs and such and specifying the breakpoint regex is impossible.

Reply:
I have restored the old behaviour by adding `filter_replay_instantiations`.